planning/behavior_path_dynamic_avoidance_module/README.md

```diff
@@ -4,15 +4,16 @@ This module is under development.
 
 ## Purpose / Role
 
-This module provides avoidance functions for vehicles, pedestrians, and obstacles in the vicinity of the ego's path in combination with the obstacle_avoidance module.
+This module provides avoidance functions for vehicles, pedestrians, and obstacles in the vicinity of the ego's path in combination with the Obstacle Avoidance module.
```

Comment:
obstacle_avoidance_planner package or module sounds more precise considering that we have obstacle avoidance module in the behavior path planner.